nixos/doc/manual/configuration/profiles/minimal.section.md

```diff
@@ -1,6 +1,6 @@
 # Minimal {#sec-profile-minimal}
 
 This profile defines a small NixOS configuration. It does not contain any
-graphical stuff. It's a very short file that sets [](#opt-i18n.supportedLocales)
```

Comment:
I thought this was a change that should be decoupled from the `visible = false;` change. I suggested earlier that it should appear in the beginning of the PR and hence should not change a lot while we improve the rest of the changes.

Reply:
> The dangling reference will cause the doc builder to fail, because `supportedLocales` is now hidden from the documentation, so it can no longer be referenced.

Fair enough. Thanks for explaining!